src/Documentation/Action/DocumentationAction.php

```diff
@@ -27,6 +27,9 @@
  */
 final class DocumentationAction
 {
+    public const OPENAPI_FORMAT = 'openapi';
+    public const JSON_FORMAT = 'json';
```

Comment:
Those constants should be declared (or maybe already exist) in a normalizer or serializer